<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>726</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1133</width>
    <height>726</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow#MainWindow
{

}
QLabel#QlabelCamera
{

}

QPushButton#pushButtonStart
{
font-size: 16px;
background-color: white;
border-color:#4CAF50;  /* Green */
border-radius: 2px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }

QPushButton#pushButtonStart:hover 
{
border-style: outset;
border-width:1.2px;
background-color: #4CAF50; /* Green */
color: white
 }
QPushButton#pushButtonStart:pressed 
{
border-style: outset;
border-width:2.2px;
background-color: #4CAF50; /* Green */
color: white
 }
QPushButton#pushButtonStop 
{
font-size: 16px;
background-color: white;
border-color:#f44336;
border-radius: 2px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }

QPushButton#pushButtonStop:hover 
{
border-style: outset;
border-width:1.2px;
background-color: #f44336; /* Green */
color: white
 }
QPushButton#pushButtonStop:pressed 
{
border-style: outset;
border-width:2.2px;
background-color: #f44336; /* Green */
color: white
 }

QPushButton#pushButtonSaveLog
{
font-size: 16px;
background-color: white;
border-color:#008CBA;
border-radius: 2px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }

QPushButton#pushButtonSaveLog:hover 
{
border-style: outset;
border-width:1.2px;
background-color: #008CBA; /* BLUE */
color: white
 }
QPushButton#pushButtonSaveLog:pressed 
{
border-style: outset;
border-width:2.2px;
background-color: #008CBA; /* BLUE */
color: white
 }

QPushButton#pushButtonClearLog
{
font-size: 16px;
background-color: white;
border-color:#f44336; 
border-radius: 2px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }


QPushButton#pushButtonClearLog:hover 
{
border-style: outset;
border-width:1.2px;
background-color: #f44336; /* RED */
color: white
 }
QPushButton#pushButtonClearLog:pressed 
{
border-style: outset;
border-width:2.2px;
background-color: #f44336; /* RED */
color: white
 }


QPushButton#pushButtonVerify
{
font-size: 16px;
background-color: white;
border-color:#555555;
border-radius: 2px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }

QPushButton#pushButtonVerify:hover 
{
border-style: outset;
border-width:1.2px;
background-color: #555555; /* Grey */
color: white
 }
QPushButton#pushButtonVerify:pressed 
{
border-style: outset;
border-width:2.2px;
background-color: #555555; /* Grey */
color: white
 }




QPushButton#pushButtonTelegram
{
font-size: 16px;
background-color: white;
border-color:#555555;
border-radius: 2px;
padding: 4px;	
border-style: inset;
border-width:1.2px;
 }
QPushButton#pushButtonTelegram:hover 
{
border-style: outset;
border-width:1.2px;
background-color: #555555; /* Grey */
color: white
 }
QPushButton#pushButtonTelegram:pressed 
{
border-style: outset;
border-width:2.2px;
background-color: #555555; /* Grey */
color: white
 }







</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="QlabelCamera">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>571</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveLog">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save logs</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogs">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logs count</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOnline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ONLINE </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOffline">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFFLINE  </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVerify">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUid">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Firebaser UID :</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetLogs">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>260</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelMode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>STATUS : OFFLINE  - MODE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUid">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelToken">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telegram Token :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTelegram">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditToken">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearLog">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear logs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionManage_user"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_user">
   <property name="text">
    <string>Manage user</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
